src/index.html

```diff
@@ -638,7 +638,7 @@ <h1 class="clear">Map boundary change</h1>
                   </div>
                 </div>
               </div>
-              <p class="point-filters__no-data hidden">No filters available for the selected data.</p>
+              <p class="point-filters__no-data hidden i18n" data-i18n="No filters available for the selected data.">No filters available for the selected data.</p>
```

Comment:
yes I was considering if we should do that also. so if you also think that's what we should do, we can do it. but if we do it now, because of [WNCM-523](https://wazimap.atlassian.net/browse/WNCM-523) this would look _more_ broken since the texts will be empty when navigating to another geography.